charts/streamsets/control-agent/2.0.101/Chart.yaml

```diff
@@ -9,6 +10,7 @@ keywords:
 - streamsets
 - sdc
 - sch
+kubeVersion: 1.18 - 1.21
```

Comment:
Based on Helm Docs this is valid: https://v3.helm.sh/docs/topics/charts/#the-kubeversion-field

`hyphen ranges for closed intervals, where 1.1 - 2.3.4 is equivalent to >= 1.1 <= 2.3.4.`